data/json/items/tool/science.json

```diff
@@ -414,6 +414,7 @@
     "weight": "22700 g",
     "volume": "6 L",
     "price": 300000,
+    "price_postapoc": 45000,
```

Comment:
```suggestion
    "price_postapoc": 45`00,
```

Reply:
Yes, you're right, this item is mostly useful for its disassembled parts or in the crafting of other items.

They are 2 pipes (10 cents each), 1 rubber hose (1 dollar), 1 tiny electric motor(1 dollar), 2 plastic gaskets (50 cents each), 1 mechanical pump (5 dollars) and 1 drive belt (5 dollars). (Total 13 dollars and 20 cents)

Of course, any item is more valuable intact than disassembled.

*all prices are post-apoc.